From 1320d5b69b22dcb6479b1b87452d81e4ac034280 Mon Sep 17 00:00:00 2001
From: Muthukumar K <muthukumar.kandasamy@intel.com>
Date: Fri, 20 Sep 2013 14:30:35 +0530
Subject: [AUDIO] Disable non linear post-processing on request from HAL

BZ: 130449

This patch adds the platform state constants for
BypassNonLinearPp states.

Change-Id: I377600477714ffe223a50006bdf405f12d881a52
Category: feature differentiation
Domain: AUDIO_MEDIA-HAL
Origin: internal
Upstream-Candidate: no, proprietary
Signed-off-by: Muthukumar K <muthukumar.kandasamy@intel.com>
Reviewed-on: http://android.intel.com:8080/133290
Reviewed-by: De Chivre, Renaud <renaud.de.chivre@intel.com>
Reviewed-by: Di Folco, Neil <neil.di.folco@intel.com>
Reviewed-by: Quintero, Jorge <jorge.quintero@intel.com>
Tested-by: Barthes, FabienX <fabienx.barthes@intel.com>
Reviewed-by: cactus <cactus@intel.com>
Tested-by: cactus <cactus@intel.com>
---
 include/hardware/audio.h |    5 +++++
 1 files changed, 5 insertions(+), 0 deletions(-)

diff --git a/include/hardware/audio.h b/include/hardware/audio.h
index 07f7f12..55e13e8 100644
--- a/include/hardware/audio.h
+++ b/include/hardware/audio.h
@@ -120,6 +120,11 @@ __BEGIN_DECLS
 #define AUDIO_PARAMETER_VALUE_CONTEXT_AWARENESS_ON "on"
 #define AUDIO_PARAMETER_VALUE_CONTEXT_AWARENESS_OFF "off"
 
+/* No Non-linear Post processing */
+#define AUDIO_PARAMETER_KEY_BYPASS_NON_LINEAR_POSTPROCESSING_SETTING "BypassNonLinearPp"
+#define AUDIO_PARAMETER_VALUE_BYPASS_NON_LINEAR_PP_ON "on"
+#define AUDIO_PARAMETER_VALUE_BYPASS_NON_LINEAR_PP_OFF "off"
+
 /**
  *  audio stream parameters
  */
-- 
1.7.4.1

